apps/methodologies/bold-carbon-organic/rule-processors/credit/nft-metadata-selection/src/lib/nft-metadata-selection.dto.ts

```diff
@@ -9,13 +9,13 @@ import type { tags } from 'typia';
 
 export interface MassMetadata {
   documentId: NonEmptyString;
-  measurementUnit: NonEmptyString;
-  originCity: NonEmptyString;
-  originCountry: NonEmptyString;
-  originCountryState: NonEmptyString;
-  recyclerName: NonEmptyString;
-  subtype: NonEmptyString;
-  type: NonEmptyString;
+  measurementUnit: string;
+  originCity: string;
+  originCountry: string;
+  originCountryState: string;
+  recyclerName: string;
+  subtype: string;
+  type: string;
```

Comment:
It is not clear to me why we had to change from `NonEmptyString` to `string`. Can you elaborate?